geostore/management/commands/import_geojson.py

```diff
@@ -27,7 +27,7 @@ def add_arguments(self, parser):
                                           "A new layer is created if not present.")
         exclusive_group.add_argument('-ln', '--layer-name',
                                      action="store",
-                                     help="Name of created layer containing GeoJSON datas."
+                                     help="Name of created layer containing GeoJSON data . "
```

Comment:
ame of created layer containing GeoJSON data.

Reply:
data and not datas